content/articles/build-nextjs-with-mongodb-and-deploy-on-vercel/index.md

```diff
@@ -0,0 +1,982 @@
+[Next.js](https://nextjs.org/) is a JavaScript framework that lets you build server-side rendering and static web applications using React.js as a client-side web-driven framework, just like vue.js and Angular.js. It has a number of features that make it great such as pre-rendering, CSS in JS, and many more.
+
+[Next.js](https://nextjs.org/) is a hybrid framework. It is the easiest way to build React.js applications. It includes a lot of functionalities that are easily extensible and easily customizable. Being a hybrid means it can be used for either CSR (Client-Side Rendering), SSR (Server-Side Rendering), SSG (Static-Site Generation). Meaning you can use databases and perform data fetching using Next.js. This makes it a Static Site Generation framework (SSG), meaning you will build pre-rendered sites that load blazingly fast.
+
+### Goal
+
+In this guide, you will learn how to create a Next.js application using MongoDB and then deploy the application on [Vercel](https://vercel.com/dashboard). We will create a simple blog application that communicates with MongoDB. This will allow you to add a new post to the blog app and update or delete a post from the app.
```

Comment:
```suggestion
In this guide, you will learn how to create a Next.js application, connect it to MongoDB and deploy it on [Vercel](https://vercel.com/dashboard). We will create a simple blog application stores blog posts in a MongoDB database. This will allow you to add a new post to the blog app and update or delete a post from the app.
```